views/sponsor.ejs

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<html>
+<head>
+    <% include template/head.ejs %>
+    <link rel="stylesheet" type="text/css" href="/css/index.css"/>
+    <style>
+        .main-footer {
+            position: absolute;
+            bottom: 0;
+            width: 100%;
+        }
+    </style>
+</head>
+
+<body>
+<% include template/nav.ejs %>
+
+<h3>&nbsp;</h3> <!-- placeholder -->
+<div class="container sponsor-container">
```

Comment:
Add `body-layout` class to make this div a page. Then you can remove the `place holder: <h3></h3>` and css for footer. Take advantage of existing css classes.
Also is `sponsor-container` used anywhere? If not remove unnecessary classes